sale_product_set/tests/test_product_set.py

```diff
@@ -166,3 +166,13 @@ def test_discount(self):
         )
         order_line.ensure_one()
         self.assertEqual(order_line.discount, set_line.discount)
+
+    def test_product_set_delete_in_wizard(self):
+        self.wizard = self._get_wiz()
```

Comment:
```suggestion
        wizard = self._get_wiz()
```
there's no need to set this on the test class instance

Reply:
Yes, I just forgot to take out from self.